### DISCUSSION TITLE
Pre-built binary for conductor-server 3.1.0?

### DISCUSSION THREAD

**Discussion by DPain** [Netflix/conductor#2380] [Q&A] [answered]
Hi, I was wondering if there was a possibility to get a pre-built binary for conductor-server 3.1.0 into the maven repo.

For example, the maven repo has `conductor-server-2.29.0-all.jar` hosted, but it doesn't have `conductor-server-3.1.0-all.jar`. It only has `boot`, `javadoc`, `sources`, `module`, and `pom`.

[Netflix Conductor's documentation](https://netflix.github.io/conductor/server/) quotes to use the jar file with the `all` classifier. Did it get changed to `boot` since Conductor 3.0?

Thanks in advance!

**Comment by apanicker-nflx** [comment, marked as answer]
@DPain Your observation is correct that the jar file has changed to `conductor-server-3.1.0-boot.jar` since 3.0 release. 
Thanks for pointing out the inconsistency in documentation and apologies for the confusion.
